docs_guidelines/how_to/crossreference.rst

```diff
@@ -0,0 +1,82 @@
+================================================
+Crossreference to other pages from documentation
+================================================
+
+This guide shows how to reference to other pages in the documentation.
+
+Pre-requisites
+==============
+
+This guide assumes your Qiskit project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+`sphinx-quickstart <https://www.sphinx-doc.org/en/master/man/sphinx-quickstart.html>`_.
+
+Add the links
+=============
+
+There are several types of pages that can be referenced:
+
+* Module pages
+* Function pages
+* Class pages
+* Method pages
+* Attribute pages
+
+Module pages
+------------
+
+If the page to which you want to link corresponds to a module, you should use ``:mod:`qiskit.module_name```.
```

Comment:
Does this line render correctly within the docs guidelines? I wouldn't think so since we don't have the apidocs from terra here. I would suggest to change these to pure text.

Another way to do it is to copy the apidoc of `qiskit.circuit` to the project. Probably that's better for showcasing the sphinx inter connect capability.

Reply:
Or we could try to use `intersphinx` for that.

Reply:
However that might be cheating, as the links take you to qiskit.org 😅